civihr_employee_portal/views/views_export/views_hr_staff_directory.inc

```diff
@@ -290,13 +300,13 @@ $handler->display->display_options['fields']['title']['field'] = 'title';
 $handler->display->display_options['fields']['title']['relationship'] = 'details_revision_id';
 $handler->display->display_options['fields']['title']['label'] = 'Job Title';
 /* Field: CiviCRM Email: Email Address */
-$handler->display->display_options['fields']['email']['id'] = 'email';
-$handler->display->display_options['fields']['email']['table'] = 'civicrm_email';
-$handler->display->display_options['fields']['email']['field'] = 'email';
-$handler->display->display_options['fields']['email']['label'] = 'Email';
-$handler->display->display_options['fields']['email']['location_type'] = '2';
-$handler->display->display_options['fields']['email']['location_op'] = '0';
-$handler->display->display_options['fields']['email']['is_primary'] = 0;
+$handler->display->display_options['fields']['email_1']['id'] = 'email_1';
```

Comment:
No problem, so long as it's consistent throughout the view, as it's the field's name within it.  The "_1" was added because at some point I had both the original field that didn't work and the new field that worked together in the view, to compare their configuration.  So kept the new one (email_1) and deleted the old one (email).